orttraining/orttraining/python/training/ortmodule/_graph_execution_manager.py

```diff
@@ -321,6 +321,16 @@ def _get_exported_model(self, input_schema: _ModelInputOutputSchemaType, *inputs
             # FlattenedModule needs _InputInfo to expand user input from *args to *args + **kwargs
             self._flattened_module._input_info = self._input_info
 
+            # Leverage cached model if available
+            cache_dir = self._runtime_options.ortmodule_cache_dir
+            cache_prefix = self._runtime_options.ortmodule_cache_prefix
+            rank = os.getenv("RANK", default="0")
```

Comment:
```suggestion
            if torch.distributed.is_initialized():
                rank = torch.distributed.get_rank()
```

Reply:
@pengwa when I test this using `torchrun --nproc_per_node 4 orttraining_test_ortmodule_api.py` it only saves 1 model despite 4 running processes. When I do the original way via `RANK` env var it correctly saves 4 models.